Documentation/Contributing/ExperimentalFeatures.md

```diff
@@ -11,7 +9,7 @@ If a component is marked experimental you can expect the following:
 - Experimental features are subject to change. 
 
 
-# How to contribute an Experimental feature
+# Experimental feature guidelines
 ### Experimental code should live in a separate folder
  Make sure to mirror the same folder structure of MRTK. [You can use the following PR as an example](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/4532). 
```

Comment:
I personally had trouble here finding a great example of how all folder types mapped. Specifically Experimental inspectors. We should probably just call out the folder structure explicitly like we do on line 6.

> An example scene demonstrating usage, located under `MixedRealityToolkit.Examples\Experimental` subfolder

Reply:
Resolved, I hope. Tried to add some examples.